Classes/Hooks/KeSearchIndexer.php

```diff
@@ -70,13 +70,35 @@ public function customIndexer(&$indexerConfig, &$indexerObject)
             $abstract = strip_tags($originalObject->getKeSearchAbstract($index));
             $content = strip_tags($originalObject->getKeSearchContent($index));
             $fullContent = $title . "\n" . $abstract . "\n" . $content;
+            $tags = [];
+            foreach ($originalObject->getCategories() as $category) {
```

Comment:
Would it make sense to add `getCategories()` to `KeSearchIndexInterface`? Or `getKeSearchTags()` - which one is better? Looks a little cleaner than `is_callable` and` instanceof Category` to me.